### ISSUE TITLE
the loss is nan

### ISSUE THREAD

**Issue by cena001plus** [hli1221/imagefusion_densefuse#14]
i have try different lr  from 0.00001  to 0.1  , but  thr loss is nan ,   why?
![image](https://user-images.githubusercontent.com/37894345/122056144-e799fe80-ce1b-11eb-92c4-4daa08fbe731.png)

**Comment by LuoXubo** [comment]
same problem, have you solved it?

**Comment by GHGluck** [comment]
I also had the same problem.
![1648638909(1)](https://user-images.githubusercontent.com/101547666/161044239-301aadd9-7e43-4e50-8142-d2f44e130524.png)

**Comment by hli1221** [comment]
Hi, thank you very much for report this problem.  
Normally, the pixel loss should be decreased with increasing iteration number. However, the pixel loss was increasing. I suggest you can check the input and the output, this problem may occur if they are not in the same domain.

**Comment by GHGluck** [comment]
Thank you for your reply. I tried to modify the input data and found that sometimes the training can be successful, but when I increase the data, the loss will still be none. I don't know if it is because the network cannot fit my training data.
